lib/multiselect_dialog.dart

```diff
@@ -53,7 +66,8 @@ class _MultiSelectDialogState<V> extends State<MultiSelectDialog<V>> {
   @override
   Widget build(BuildContext context) {
     return AlertDialog(
-      title: Text(widget.title),
+      title: widget.title,
```

Comment:
Having this referenced from outside gives more control for customization as and when required